text_reading/src/main/scala/org/clulab/aske/automates/alignment/Aligner.scala

```diff
@@ -15,7 +22,7 @@ trait Aligner {
 }
 
 
-class VariableEditDistanceAligner(relevantArgs: Set[String]) {
+class VariableEditDistanceAligner(relevantArgs: Set[String] = Set("variable")) {
```

Comment:
@maxaalexeeva -- seeing this is triggering a memory for me -- there was some function or something where I think at one point in the API refactoring the relevant Args wasn't getting populated right, did I remember to fix it?  Basically, is it working the way we expect for both comment extractions and text extractions?  how can we check?

Reply:
I'll look into this. Thanks!